docs/commands/logic-gates.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Execute Logic Gates
+category: Techniques
+mentions:
+    - BedrockCommands
+    - zheaEvyline
+nav_order: 1
+---
+
+## Introduction
+
+[Sourced By Bedrock Commands Community Discord](https://discord.gg/SYstTYx5G5)
+
+Logic gates are devices that take one or more binary inputs and, as the name suggests, perform logical boolean operations to give a single binary output (either `true` or `false`).
+
+They are what a computer uses to perform any function. These devices are very helpful for programming, and using them is a crucial step in creating optimized command systems. So, let's learn how you can use any of the logic gates in your systems using the `/execute` command.
+
+
+## Logic Gates
+
+**Chart:**
+
+![logicGatesTable](/assets/images/commands/logicGatesTable.jpg)
+
+**Converted To Commands:**
+
+- Buffer:
+  - `/execute if entity @s [tag=red] run <command>`
+- (Inverter) NOT Gate:
+  - `/execute if entity @s [tag=!red] run <command>`
+- AND Gate:
+  - `/execute if entity @s [tag=red, tag=green] run <command>`
+- (NOT AND) NAND Gate:
+  - `/execute unless entity @s [tag=red, tag=green] run <command>`
+- OR Gate:
+  - `/execute unless entity @s [tag=!red, tag=!green] run <command>`
+- NOR Gate:
+  - `/execute if entity @s [tag=!red, tag=!green] run <command>`
+- XOR Gate:
+  - `/execute unless entity @s [tag=!red, tag=!green] unless entity @s [tag=red, tag=green] run <command>`
+- XNOR Gate:
+  - `/execute unless entity @s [tag=red, tag=!green] unless entity @s [tag=!red, tag=green] run <command>`
+
+## Explanation
+
+*Explanation provided by @Champ0401:*
+
+**AND Gate:**
+- `/execute as @p [tag=red, tag=green] run say success`
+
+The AND gate takes at least two inputs. In this case, if the player meets both requirements (meaning they have both the tag red and the tag green), the command will run. The AND gate is the most commonly used gate in Minecraft commands.
```

Comment:
`meaning they have both the tag red and the tag green` -> having the "red" and "green tag"